packages/ckeditor5-table/src/tablecolumnresize/utils.js

```diff
@@ -365,3 +365,21 @@ export function removeColumnResizerElements( viewWriter, viewCell ) {
 
 	viewWriter.remove( viewTableColumnResizerElement );
 }
+
+// Shows/hides resizers in specified view.
+//
+// @param {module:ui/view~View} view
+// @param {true|false} value
```

Comment:
```suggestion
// @param {Boolean} value
```

It's a boolean value - just document its type.

Reply:
Also I think that a value that tells the purpose would be better, e.g. `isVisible`.

Reply:
True... ;)